fhir-term-graph/src/test/java/com/ibm/fhir/term/graph/test/FHIRTermGraphTest.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * (C) Copyright IBM Corp. 2020, 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.term.graph.test;
+
+import static com.ibm.fhir.term.graph.util.FHIRTermGraphUtil.toObject;
+
+import java.util.stream.Stream;
+
+import org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.GraphTraversal;
+import org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.GraphTraversalSource;
+import org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.__;
+import org.apache.tinkerpop.gremlin.process.traversal.step.filter.TimeLimitStep;
+import org.apache.tinkerpop.gremlin.structure.Vertex;
+
+import com.ibm.fhir.model.type.Code;
+import com.ibm.fhir.model.type.Decimal;
+import com.ibm.fhir.term.graph.FHIRTermGraph;
+import com.ibm.fhir.term.graph.factory.FHIRTermGraphFactory;
+
+public class FHIRTermGraphTest {
+    @SuppressWarnings({ "unchecked", "rawtypes" })
+    public static void main(String[] args) throws Exception {
+        FHIRTermGraph graph = FHIRTermGraphFactory.open("conf/janusgraph-berkeleyje-lucene.properties");
+
+        GraphTraversalSource g = graph.traversal();
+
+        g.V().drop().iterate();
+//      g.E().drop().iterate();
```

Comment:
```suggestion
//      g.E().drop().iterate();
```

Intended?

Reply:
Only vertices need to be dropped. Edges are dropped automatically. Will removed the commented line.